### PR TITLE
Services 1959 add marketplace disable state

### PR DESCRIPTION
**Pull request by danielailie**

### CODE REVIEW

File: src/modules/assets/assets-getter.service.ts [hunk 1]
```diff
@@ -65,19 +66,19 @@ export class AssetsGetterService {
...
-      if (!ticketsCollections || ticketsCollections?.length === 0) return new CollectionType<Asset>();
-
-      const ticketCollectionIdentifiers = ticketsCollections.map((x) => x.collection).toString();
-      return await this.getAssetsForUser(
-        filters.ownerAddress,
-        `?identifiers=${ticketCollectionIdentifiers}&from=${offset}&size=${limit}`,
-        `?identifiers=${ticketCollectionIdentifiers}`,
-      );
-    }
+    // if (filters.ownerAddress && filters.customFilters) {
```
Comment (claudiulataretu) [inline src/modules/assets/assets-getter.service.ts:69]: commented code

File: src/modules/rabbitmq/blockchain-events/disable-marketplace/disable-marketplace-events.service.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { MarketplaceTypeEnum } from 'src/modules/marketplaces/models/MarketplaceType.enum';
+import { Marketplace } from 'src/modules/marketplaces/models';
+import { MarketplaceEntity } from 'src/db/marketplaces';
+
+@Injectable()
+export class DisabledMarketplaceEventsService {
+  constructor(private redisCacheService: RedisCacheService, private readonly marketplaceEventsService: MarketplaceEventsService) {}
+
+  public async handleAuctionEventsForDisableMarketplace(auctionEvents: any[], hash: string) {
+    if (auctionEvents?.length) {
```
Comment (claudiulataretu) [inline src/modules/rabbitmq/blockchain-events/disable-marketplace/disable-marketplace-events.service.ts:14]: check for length > 0

File: src/modules/rabbitmq/blockchain-events/disable-marketplace/disable-marketplace-events.service.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      await this.redisCacheService.rpush(CacheInfo.MarketplaceEvents.key, JSON.stringify([{ hash: hash, events: auctionEvents }]));
+    }
+  }
+
+  public async processMissedEventsSinceDisabled(marketplaces: MarketplaceEntity[]) {
+    const events = await this.redisCacheService.lpop(CacheInfo.MarketplaceEvents.key);
+    if (events?.length) {
```
Comment (claudiulataretu) [inline src/modules/rabbitmq/blockchain-events/disable-marketplace/disable-marketplace-events.service.ts:21]: check for length > 0

File: src/modules/rabbitmq/blockchain-events/nft-events.module.ts [hunk 4]
```diff
@@ -84,7 +86,7 @@ import { MintersModuleGraph } from 'src/modules/minters/minters.module';
     MinterEventsService,
     RevertEventsConsumer,
     RevertEventsService,
-    ElasiticUpdatesConsumer,
+    // ElasiticUpdatesConsumer,
```
Comment (claudiulataretu) [inline src/modules/rabbitmq/blockchain-events/nft-events.module.ts:89]: commented code

File: src/modules/rabbitmq/blockchain-events/disable-marketplace/disable-marketplace-events.service.ts (hunk 2 above)
Comment (danielailie) [reply src/modules/rabbitmq/blockchain-events/disable-marketplace/disable-marketplace-events.service.ts]: done

File: src/modules/rabbitmq/blockchain-events/nft-events.module.ts (hunk 4 above)
Comment (danielailie) [reply src/modules/rabbitmq/blockchain-events/nft-events.module.ts]: done

**Review by claudiulataretu — APPROVED** [review]

**Review by gabrielmatei — APPROVED** [review]

**Pull request merged**